packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -272,21 +270,19 @@ export class MessageWidget extends Component {
     const handleEvent = this.handleEvent;
     spark.mercury.on(`event:conversation.activity`, (event) => {
       const activity = event.data.activity;
-      const isVisibleContent = VISIBLE_ACTIVITY_VERBS.indexOf(activity.verb) !== -1
-        && VISIBLE_ACTIVITY_TYPES.indexOf(activity.object.objectType) !== -1;
       // Ignore activity from other conversations
       if (activity.target.id === conversationId) {
-        if (isVisibleContent) {
+        if (activity.object.objectType === `activity`) {
+          props.receiveMercuryActivity(activity);
+        }
+        else {
```

Comment:
oooo nice! Turns out if you filter things before hand you don't have to check it over and over again!